htdocs/modules/system/class/cookie.php

```diff
@@ -15,7 +15,7 @@
  * @author              Andricq Nicolas (AKA MusS)
  * @package             system
  */
-class cookie
+class Cookie
```

Comment:
Is this class actually used? I'm not seeing it anywhere. (not a problem, just a question. Thought you might know :smile: )